test/edu/stanford/dlss/was/TestWasapiDownloader.java

```diff
@@ -9,18 +11,45 @@
 
 public class TestWasapiDownloader {
 
-    @Before
```

Comment:
nearly all the other java files are 2 spaces indent.  I was just making it consistent.  @jmartin-sul ... what say you?